README.md

```diff
@@ -9,6 +9,40 @@ Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 This is a NPM module to aid with embedding the Quill editor in [Stripes](https://github.com/folio-org/stripes-core/) applications for building templates with token substitution.
 
+
+## Value Sanitization
+
+In any case where a user-created HTML string will be rendered directly to the UI, it should be sanitized to eliminate any issues with malformed tags/attributes.This library exports a `sanitize` function that should be used within the ui-module prior to passing the value to the form. The function accepts the value to be rendered and an optional overriding configuration for `DOMPurify` - the sanitization library.
```

Comment:
Pedantic: `s/attributes.This/attributes. This/`. 

Maybe change the last sentence(s) to, "The function accepts the value to be rendered and an optional overriding configuration for the sanitization library (currently `DOMPurify`). It returns a sanitized copy of its input." This further de-canonizes DOMPurify as The One True Implementation, and emphasizes that we know how to use immutable data, which isn't really a problem with strings, but I will eternally feel the burn of `Array.sort()`.